### DISCUSSION TITLE
launcher button not working?

### DISCUSSION THREAD

**Discussion by 998amarithetopg** [linuxdeepin/developer-center#6887] [Troubleshooting | 故障排查] [answered]
i use arch btw but the button does not work trying command dont work

**Comment by 998amarithetopg** [comment]
and how can i fix it ?

**Reply by BLumia** [reply]
Are you using `deepin-launcher` or `deepin-launchpad`? If you are using `deepin-launcher`, try install `deepin-launchpad` instead and try again. If it still isn't working, launch `dde-launchpad` from terminal and provide its log. Thanks!

**Reply by 998amarithetopg** [reply]
I think deep in-launcher or launchpad

**Reply by 998amarithetopg** [reply]
Still does not work

**Reply by BLumia** [reply]
> deep in-launcher or launchpad

Only one can exist at the same time. Please confirm which one is the one you're using.

> Still does not work

Please provide the log I asked before. Thanks!

**Reply by 998amarithetopg** [reply]
its launchpad and the log is   [amaria@archlinux ~]$ dde-launchpad
register dbus service failed
26
"" 1
dtk.core.dsg: AppId is fetched from AM, and value is  "deepin-terminal"
DGuiApplicationHelper: Don't use QGuiApplication::setPalette on DTK application.
createPlatformWindow QWindow(0x7ffffd6610a0) Qt::Window QWindow(0x0)
setEnableNoTitlebar true DialogWindow_QMLTYPE_86(0x560e55f1f5b0) Qt::Window QWindow(0x0)
setEnableNoTitlebar true Main_QMLTYPE_95(0x560e55ce8990) Qt::Window QWindow(0x0)
qrc:/qml/Main.qml:205:17: QML WarningButton: Binding loop detected for property "palette.windowText"
createPlatformWindow DialogWindow_QMLTYPE_86_QML_97(0x560e55f1f5b0) Qt::Dialog QWindow(0x0)
createPlatformWindow Main_QMLTYPE_95(0x560e55ce8990) Qt::Tool QWindow(0x0)
Got wallpaper URL from dbus: QUrl("file:///usr/share/deepin-themes/organic-glass/organic-glass-dark.jpg")

**Reply by 998amarithetopg** [reply]
After reboot it was fixed will upload picture

**Reply by BLumia** [reply]
Oh, sorry I forget, `dde-launchpad` by default will only run in the background (to speedup launch time). It needs a `-s` argument to show UI directly. Otherwise, you still need to click the launcher icon to bring up the UI.

Good to hear your issue get resolved after reboot :)

**Comment by 998amarithetopg** [comment, marked as answer]
![Uploading image.jpg…]()
